roofit/roofitcore/inc/RooGradMinimizerFcn.h

```diff
@@ -0,0 +1,92 @@
+/*****************************************************************************
+ * Project: RooFit                                                           *
+ * Package: RooFitCore                                                       *
+ * @(#)root/roofitcore:$Id$
+ * Authors:                                                                  *
+ *   AL, Alfio Lazzaro,   INFN Milan,        alfio.lazzaro@mi.infn.it        *
+ *   PB, Patrick Bos,     NL eScience Center, p.bos@esciencecenter.nl        *
+ *   VC, Vince Croft,     DIANA / NYU,        vincent.croft@cern.ch          *
+ *                                                                           *
+ *                                                                           *
+ * Redistribution and use in source and binary forms,                        *
+ * with or without modification, are permitted according to the terms        *
+ * listed in LICENSE (http://roofit.sourceforge.net/license.txt)             *
+ *****************************************************************************/
+
+#ifndef ROO_GRAD_MINIMIZER_FCN
+#define ROO_GRAD_MINIMIZER_FCN
+
+#include <vector>
```

Comment:
RooFit first, rest of ROOT middle, `vector` last

Reply:
The Google style guide recommends to first put standard library includes. I don't want to start a debate here, of course, so I'll change it here, but just wondering: why the standard include last?

Reply:
Good! The google style guide is of course an option, but they probably have "include what you use" checkers. We don't, so inverting the order is more strict.